printing/pptactic.ml

```diff
@@ -1200,6 +1202,8 @@ module Make
             pr_may_eval pr.pr_constr pr.pr_lconstr pr.pr_constant pr.pr_pattern c
           | TacFreshId l ->
             keyword "fresh" ++ pr_fresh_ids l
+          | TacExactId x ->
+            primitive "exact_id" ++ pr_id x
```

Comment:
Well ``TacExactId`` is an AST entry in tactics that when interpreted results in the exact given identifier. It is perhaps not that general but I could imagine that it could be used for other tactics implemented in ML to produce an identifier. So "ident_of_string" might not be that suitable. But I understand that this "exact_id" is not very informative to the user!